.github/workflows/build_and_test.yml

```diff
@@ -168,6 +168,32 @@ jobs:
       - name: Test Tutorials
         run: cd build_examples/tutorials_bin && ./Tutorial-HelloWorld
 
+  freebsd:
```

Comment:
Why do we need FreeBSD in GitHub Actions? We already run it in CirrusCI...

Reply:
I plan to use it allow for release when we need to pack and upload an artifact

Reply:
If this is just for release, maybe we don't need to run all the tests?

My main concern is that it runs on a macOS machine and the limit is much lower compares to Linux machines:
https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits

So if the build runs for a long time it means other macOS builds need to wait in the queue and the overall build time goes up

Reply:
Yes this should be only used for packaging, I will move this to a separate pipeline without any test